rest/poly_collection.cc

```diff
@@ -51,6 +52,8 @@ PolyConfigDescription()
     addField("persistent", &PolyConfig::persistent,
              "If true, then this element will have its configuration stored "
              "and will be reloaded on startup", false);
+    addField("deterministic", &PolyConfig::deterministic,
+             "If true, the entity has no hidden state ", true);
```

Comment:
This one is for all entities, FunctionPolyConfigDescription is specifically for functions, and functions *per definition* are not supposed to have internal states :)